docs/streams/upgrade-guide.html

```diff
@@ -44,6 +44,20 @@
         See <a href="#streams_api_changes_0101">below</a> a complete list of 0.10.1 API changes that allow you to advance your application and/or simplify your code base, including the usage of new features.
     </p>
 
+    <h3><a id="full_upgrade_workflow" href="#full_upgrade_workflow">Full Upgrade Workflow</a></h3>
```

Comment:
Do we want to recommend this pattern for all the upgrade paths though, as it is not a pleasant path compared with rolling bounces (i.e. your app will be offline for a bit); I think for most of the upgrade path rolling bounce is still possible?

Reply:
I agree with Guozhang. We can leave this PR open for now and should look into our upgrade system test -- after we verify what work, we should update the docs accordingly. I think, this whole "stop the app completely" thing, is only required if you upgrade from `0.10.0` -- in general, rolling bounce upgrades should work.

Reply:
Ack